Allow setting keyboard brightness on MacbookPro11,4.

MegaManSec opened the pull request:
Some systems such as the MacbookPro11,4 allow setting of the keyboard backlight using dev.asmc.0.light.control, but not the monitor backlight. asmc would bork early when attempting to set the keyboard backlight, because the lcd backlight values were unavailable.

Example of previous behavior:

```
$ /usr/local/bin/asmctl key up
sysctl hw.acpi.video.lcd0.levels : Operation not supported by device
# sysctl dev.asmc.0.light.control=100
dev.asmc.0.light.control: 0 -> 100
```

yuichiro-naito commented:
Ah, OK. Your patch does the right thing. I'll merge this PR.
Thanks for your contribution!

MegaManSec committed:
Allow setting keyboard brightness on MacbookPro11,4.

Some systems such as the MacbookPro11,4 allow setting of the keyboard backlight using dev.asmc.0.light.control, but not the monitor backlight.
asmc would bork early when attempting to set the keyboard backlight, because the lcd backlight values were unavailable.

Example of previous behavior:

$ /usr/local/bin/asmctl key up
sysctl hw.acpi.video.lcd0.levels : Operation not supported by device
dev.asmc.0.light.control: 0 -> 100
